tests/webspec.js

```diff
@@ -1,8 +1,9 @@
 'use strict'
 const expect = chai.expect;
 
+describe('Game Function Tests', function(){
```

Comment:
Any reason you switched out of ES6 syntax?

Reply:
I can change the top one back to using arrow functions.

for the second describe I needed it for setting the time out
https://mochajs.org/#arrow-functions
Passing arrow functions (aka "lambdas") to Mocha is discouraged. Lambdas lexically bind this and cannot access the Mocha context. For example, the following code will fail: